samples/dnn/person_reid.cpp

```diff
@@ -306,14 +306,20 @@ int main(int argc, char **argv)
             }
             if (imgWidth == -1){
                 imgWidth = min(image.rows, image.cols);
-                fontSize = (stdSize*imgWidth)/stdImgSize;
-                fontWeight = (stdWeight*imgWidth)/stdImgSize;
+                fontSize = min(fontSize, (stdSize*imgWidth)/stdImgSize);
+                fontWeight = min(fontWeight, (stdWeight*imgWidth)/stdImgSize);
```

Comment:
because of this we still have different font size and thickness, depending on the video resolution. Please, make the size and the weight the same.